UPGRADE.rst

```diff
@@ -58,20 +58,62 @@ Config options
 **Note: Registration by email address or phone number will not work in this release unless
 some config options are changed from their defaults.**
 
-This is due to Synapse v1.4.0 now defaulting to sending registration and password reset tokens
-itself. This is for security reasons as well as putting less reliance on identity servers.
-However, currently Synapse only supports sending emails, and does not have support for
-phone-based password reset or account registration. If Synapse is configured to handle these on
-its own, phone-based password resets and registration will be disabled. For Synapse to send
-emails, the ``email`` block of the config must be filled out. If not, then password resets and
-registration via email will be disabled entirely.
-
-This release also deprecates the ``email.trust_identity_server_for_password_resets`` option and
-replaces it with the ``account_threepid_delegates`` dictionary. This option defines whether the
+Previous versions of Synapse delegate the sending of registration emails and
+SMS to an identity server by default. In most cases this server is vector.im or
+matrix.org.
+
+In Synapse 1.4, for security and privacy reasons, the homeserver will no longer
+delegate email or SMS to an identity server by default and instead the admin
+will need to explicitly decide how they would like email and SMS to be sent.
+
+In the medium term the vector.im and matrix.org identity servers will disable
+sending email and SMS entirely, however in order to ease the transition they
+will retain the capability to send email and SMS for a limited period. Email
+will be disabled on XXX (giving 2 months notice), disabling SMS will follow
+some time after that once SMS sending support lands in Synapse.
+
+Once email and SMS support has been disabled in the vector.im and matrix.org
+identity servers, all Synapse versions that depend on those instances will be
+unable to send email and SMS through them.
```

Comment:
Will identity servers continue to provide registration functionality in the long term or does the new implementation signal a shift towards all homeservers implementing their own? A note about this might help sysadmins choose whether to keep using their own IS or to rely on Synapse sooner rather than later.

Reply:
Good point, will do